modules/core/api/src/main/kotlin/com/wipdev/eWiLL_backend/database/tables/Task.kt

```diff
@@ -2,53 +2,36 @@ package com.wipdev.eWiLL_backend.database.tables
 
 import com.wipdev.eWiLL_backend.MediaType
 import com.wipdev.eWiLL_backend.eval.FeedbackLevel
-import org.hibernate.Hibernate
 import javax.persistence.*
+import org.hibernate.Hibernate
 
 @Entity
 @Table(name = "Task")
 data class Task(
-
-    @Id
-    @GeneratedValue(strategy = GenerationType.SEQUENCE)
-    @Column(name = "id", nullable = false)
-    var id: Long? = null,
-
-    @Column(name = "name", nullable = false)
-    var name: String? = null,
-
-    @Column(name = "description", nullable = false, length = 100000)
-    var description: String? = null,
-
-    @Column(name = "dueDate", nullable = false)
-    var dueDate: String? = null,
-
-    @Enumerated(EnumType.STRING)
-    @Column(name = "mediatype", nullable = false)
-    var mediaType: MediaType? = null,
-
-    @Enumerated(EnumType.STRING)
-    @Column(name = "liability")
-    var eLiability: ELiability? = ELiability.NONE,
-
-    @Column(name = "courseId", nullable = false)
-    var courseId: Long? = null,
-
-    @Column(name = "solutionModelId", nullable = true)
-    var solutionModelId: Long? = null,
-
-    @Column(name = "rulesetId", nullable = false)
-    var rulesetId: Long? = null,
-
-    @Column(name = "maxSubmissions", nullable = true)
-    var maxSubmissions: Int? = 999,
-
-    @Column(name = "showLevel")
-    @Enumerated(EnumType.STRING)
-    var showLevel: FeedbackLevel? = FeedbackLevel.BASIC,
+        @Id
+        @GeneratedValue(strategy = GenerationType.SEQUENCE)
+        @Column(name = "id", nullable = false)
+        var id: Long? = null,
+        @Column(name = "name", nullable = false) var name: String? = null,
+        @Column(name = "description", nullable = false, length = 100000)
+        var description: String? = null,
+        @Column(name = "dueDate", nullable = false) var dueDate: String? = null,
+        @Enumerated(EnumType.STRING)
+        @Column(name = "mediatype", nullable = false)
+        var mediaType: MediaType? = null,
+        @Enumerated(EnumType.STRING)
+        @Column(name = "liability")
+        var eLiability: ELiability? = ELiability.NONE,
+        @Column(name = "courseId", nullable = false) var courseId: Long? = null,
+        @Column(name = "solutionModelId", nullable = true) var solutionModelId: Long? = null,
+        @Column(name = "rulesetId", nullable = false) var rulesetId: Long? = null,
+        @Column(name = "maxSubmissions", nullable = true) var maxSubmissions: Int? = 999,
+        @Column(name = "showLevel")
+        @Enumerated(EnumType.STRING)
+        var showLevel: FeedbackLevel? = FeedbackLevel.BASIC,
+        @Column(name = "level", nullable = true) var level: String? = null,
```

Comment:
TaskLevel should be defined like "FeedbackLevel" with a nullable option
Use English identifiers for the code, use German only for the shown text in the frontend.